src/Microsoft.DotNet.Wpf/src/Shared/MS/Internal/SecurityHelper.cs

```diff
@@ -260,15 +228,7 @@ internal static bool CallerHasUserInitiatedNavigationPermission()
         ///  Critical: Returns a permission object, which can be misused.
         ///</SecurityNote>
         [SecurityCritical]
-        internal static CodeAccessPermission CreateUserInitiatedNavigationPermission()
-        {
-            if(_userInitiatedNavigationPermission == null)
-            {
-                _userInitiatedNavigationPermission = new UserInitiatedNavigationPermission();
-            }
-            return _userInitiatedNavigationPermission;
-        }
-        static UserInitiatedNavigationPermission _userInitiatedNavigationPermission = null;
+        internal static CodeAccessPermission CreateUserInitiatedNavigationPermission() { return default(CodeAccessPermission); }
```

Comment:
I see two places where this is called - both of which trivial to remove `CreateUserInitiatedNavigationPermission` from. They are of this form:

```C#
try
{
   CreateUserInitiatedNavigationPermission();
   DoStuff();
}
catch(SomeException e)
{
    HandleException(e);
}
finally
{
    CodeAccessPermission.RevertAssert();
}
```

You can rewrite these as follows, and delete this method.

```C#
try
{
   DoStuff();
}
catch(SomeException e)
{
    HandleException(e);
}
```